### PR TITLE
docs: improve readability

### LINKED ISSUE #2882: docs: Readability of manual.md
Found an inconsistent capitilisation of the word deno in [website/manual.md](https://github.com/denoland/deno/blob/master/website/manual.md) and a couple of general readability issues

### PR DESCRIPTION
**Pull request by andrewhughes101**
Fixes #2882 
small general improvement to manual.md to fix general readability

### CODE REVIEW

File: website/manual.md [hunk 1]
```diff
@@ -1155,13 +1155,13 @@ We are very concerned about making mistakes when adding new APIs. When adding an
...
 
-As an example, see how `Deno.rename()` was proposed and added in
+As an example, see how `deno.rename()` was proposed and added in
 [PR #671](https://github.com/denoland/deno/pull/671).
 
 ### Documenting APIs
 
 It is important to document public APIs and we want to do that inline with the
-code. This helps ensure that code and documentation are tightly coupled
+code. This helps ensure that both code and documentation are tightly coupled
```
Comment (nayeemrmn) [inline website/manual.md]: I don't think I agree with this one. Maybe give it a reread?

Comment (andrewhughes101) [reply website/manual.md]: Agreed I'll revert this bit

**Comment by lucacasonato** [comment]
I think you should run ./tools/test_format.py. Afaik the CI is failing because stuff isn't formatted correctly.

File: website/manual.md [hunk 2]
```diff
@@ -1155,7 +1155,7 @@ We are very concerned about making mistakes when adding new APIs. When adding an
 Op to Deno, the counterpart interfaces on other platforms should be researched.
 Please list how this functionality is done in Go, Node, Rust, and Python.
 
-As an example, see how `Deno.rename()` was proposed and added in
+As an example, see how `deno.rename()` was proposed and added in
```
Comment (kitsonk) [inline website/manual.md]: No, it is `Deno.rename()`, not `deno.rename()`.

Comment (andrewhughes101) [reply website/manual.md]: Your right sorry, I just looked at the name in the original issue header should have doubled checked

**Pull request merged**